dev-libs/ferrisloki/files/ferrisloki-3.0.13-r3-configure-libsigc.patch

```diff
@@ -0,0 +1,66 @@
+Rollup of src_prepare() from previous revisions plus additional fixes
+
+ * derice damn configure script
+ * fix building with libsgc++-2.6
+ * fix building with /bin/sh -> dash
+
+--- a/configure
++++ b/configure
+@@ -15228,7 +15228,7 @@ have_stlport=no
+ if test "${with_stlport+set}" = set; then :
+   withval=$with_stlport; STLPORT_CFLAGS=" -I$withval/stlport "
+ 	 STLPORT_LDFLAGS=" -L$withval/lib "
+- 	 STLPORT_LIB=" -lstlport_gcc -lpthread "
++ 	 STLPORT_LIB=" -lstlport -lpthread "
+ 	 STLPORT_LIBS=" -L$withval/lib ${STLPORT_LIB} "
+ 	 stlport_try_trivial_compile=yes
+
+@@ -15540,10 +15540,10 @@ fi
+ 		stlport_installed_version=`stlport-config --version`
+
+ 		# Calculate the available version number
+-		f_tmp=( `echo $stlport_installed_version | sed 's/[^0-9]\+/ /g'` )
++		f_tmp=" `echo $stlport_installed_version | sed 's/[^0-9]\+/ /g'` "
```

Comment:
```suggestion
+		f_tmp=" $(echo $stlport_installed_version | sed 's/[^0-9]\+/ /g') "
```

Reply:
applied, actually had the same thought, but initially wanted to make the smallest possible change that would work.